### PR TITLE
Mark/new puzzle player master

### PR DESCRIPTION
**Pull request by Ant1ng2**

### CODE REVIEW

**Comment by Ant1ng2** [comment]
Why does the playPuzzles have a move argument instead of just using generateMoves?

File: puzzlesolver/puzzles/_models/puzzle.py [hunk 1]
```diff
@@ -167,6 +167,26 @@ def generateSolutions(self):
         """
         return []
 
+    #################################################################
+    # Player methods
+    #################################################################
+
+    def playPuzzle(self, moves):
```
Comment (Ant1ng2) [inline puzzlesolver/puzzles/_models/puzzle.py:174]: Why does the playPuzzles have a move argument instead of just using generateMoves?

Comment (mpresten) [reply puzzlesolver/puzzles/_models/puzzle.py:174]: After merging, I'll fix the functionality for this.

File: puzzlesolver/puzzles/pegSolitaire/pegSolitaire.py [hunk 2]
```diff
@@ -34,34 +40,57 @@ def __init__(self, **kwargs):
...
 
+    def __str__(self, **kwargs):
+        return str(self.board)
+
     @property
     def variant(self):
         return "Triangle"
 
     ### _________ Print Funcs _______________
     def printInfo(self):
```
Comment (Ant1ng2) [inline puzzlesolver/puzzles/pegSolitaire/pegSolitaire.py:51]: The printInfo function should be returning a string, not printing the output

**Pull request merged**